source/upgrade-guide/upgrading-central-components.rst

```diff
@@ -56,35 +57,91 @@ In the case Wazuh is installed in a multi-node cluster configuration, repeat the
 Upgrading the Wazuh indexer
 ---------------------------
 
-.. note::
+The Wazuh indexer cluster remains operational throughout the upgrade. The rolling upgrade process allows nodes to be updated one at a time, ensuring continuous service availability and minimizing disruptions. The steps detailed in the following sections apply to both single-node and multi-node Wazuh indexer clusters.
 
-   Note that this upgrade process doesn't update plugins installed manually. Outdated plugins might cause the upgrade to fail.
+Preparing the Wazuh indexer cluster for upgrade
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-   To ensure compatibility with the latest Wazuh indexer and Wazuh dashboard, please update manually installed plugins accordingly. For additional information, check the `distribution matrix <https://github.com/wazuh/wazuh-packages/tree/v|WAZUH_CURRENT|#distribution-version-matrix>`__.
+Perform the following steps on any of the Wazuh indexer nodes replacing ``<WAZUH_INDEXER_IP_ADDRESS>``, ``<USERNAME>``, and ``<PASSWORD>``.
 
-The cluster remains available throughout the upgrading process in a Wazuh indexer cluster with multiple nodes. This rolling upgrade allows for the shutting down of one Wazuh indexer node at a time for minimal disruption of service.
+#. Disable shard replication to prevent shard replicas from being created while Wazuh indexer nodes are being taken offline for the upgrade.
 
-Repeat the following steps for every Wazuh indexer node replacing ``<WAZUH_INDEXER_IP_ADDRESS>``, ``<USERNAME>``, and ``<PASSWORD>``.
+   .. code-block:: bash
 
-#. Disable shard allocation.
+      curl -X PUT "https://<WAZUH_INDEXER_IP_ADDRESS>:9200/_cluster/settings" \
+      -u <USERNAME>:<PASSWORD> -k -H "Content-Type: application/json" -d '{
+         "persistent": {
+            "cluster.routing.allocation.enable": "primaries"
+         }
+      }'
 
-   .. code-block:: bash
 
-      curl -X PUT "https://<WAZUH_INDEXER_IP_ADDRESS>:9200/_cluster/settings"  -u <USERNAME>:<PASSWORD> -k -H 'Content-Type: application/json' -d'
+   **Output:**
+
+   .. code-block:: json
+
       {
-        "persistent": {
-          "cluster.routing.allocation.enable": "primaries"
-        }
+         "acknowledged" : true,
+         "persistent" : {
+            "cluster" : {
+               "routing" : {
+                  "allocation" : {
+                     "enable" : "primaries"
+                  }
+               }
+            }
+         },
+         "transient" : {}
       }
-      '
 
```

Comment:
- Command line prompt symbol missing
- Three-space tabulation should nopt apply inside codeblocks